WcaOnRails/app/views/shared/_user_avatar.html.erb

```diff
@@ -1,3 +1,4 @@
+<%# React migration of this file is partially done and is available in components/UserAvatar.js %>
```

Comment:
What's missing?

Reply:
1. `navigation.html.erb`: I tried using the react component, but styling didn't work properly. I hope somebody expert in css will be able to do it some day.
2. `users/edit.html.erb`: Again I remember some styling issue which I couldn't fix.
3. `shared/_user`: Here also faced some styling issue. Also, shared/_user is used in two places: one in delegate page (but it is changed in delegate migration page) and another in select_nearby_delegate. In select_nearby_delegate too I faced styling issue.

Reply:
I have migrated the reamining usages including some minor CSS fixes.

Reply:
Thanks a lot, @gregorbg for doing it.